README.rst

```diff
@@ -27,53 +119,73 @@ To install this module you have to install some components:
 - MySQL 5.6.14 or above (http://dev.mysql.com/downloads/mysql/)
 - gcc-c++ and gcc libraries
 
+This module also needs the installation of these other components:
+
+- Fiware-facts module installed (https://github.com/telefonicaid/fiware-facts)
+- A running instance of Orion Context Broker (https://github.com/telefonicaid/fiware-orion)
+- fiware-monitoring connected to the Orion instance to provide
+  information about servers (https://github.com/telefonicaid/fiware-monitoring).
+
+`Top`__.
+
+__ `FIWARE Policy Manager GE - Cloto`_
 
 Installation
-============
+------------
 
 Once you have all prerequisites installed, you must create a DB named cloto in your MySQL server.
-Ensure your mysql path is in your path. If not, you can add executing (change /usr/local/ with your mysql folder):
-    export PATH=$PATH:/usr/local/mysql/bin
+Ensure your mysql path is in your path. If not, you can add executing (change ``/usr/local/`` with your mysql folder):
+
+``export PATH=$PATH:/usr/local/mysql/bin``
 
 In addition, be sure you have installed mysql-devel package for development of MySQL applications.
 You should be able to install it from yum or apt-get package managers.
+
     examples: yum install mysql-devel
               apt-get install mysql-devel
               ...etc
 
```

Comment:
This is rendered as a term-definition list. Why not using the same "command line" style as before?

    centos$ sudo yum install mysql-devel
    ubuntu$ sudo apt-get install mysql-devel